### PR TITLE
fix(darkmodetoggle): tooltip adaptive to current mode

### PR DESCRIPTION
**Pull request by ThibaultNocchi**

### CODE REVIEW

File: locales/en-US.json [hunk 1]
```diff
@@ -252,7 +252,8 @@
   "title": "Title",
   "tooltips": {
     "changeLanguage": "Language",
-    "toggleDarkMode": "Toggle dark mode"
+    "darkMode": "Dark mode",
```
Comment (ferferga) [inline locales/en-US.json:255]: What about "Switch to X mode"?

Comment (ThibaultNocchi) [reply locales/en-US.json:255]: I think it's not needed, I checked the first similar example I had at hand (Flood) and it only said "dark theme" "light theme"

Comment (ThibaultNocchi) [reply locales/en-US.json:255]: But that's really a preference haha

Comment (ferferga) [reply locales/en-US.json:255]: Imo it's clearer and 100% explicit, without place to imagination.

If an user it's in dark mode and see then tooltip as "Light mode" he/she might wonder what mode it's currently in. For a lot of people this is obvious but maybe not so much for elderly or color blind :).

With switch there's no possible doubt.

Comment (ThibaultNocchi) [reply locales/en-US.json]: Done

**Review by ferferga — APPROVED** [review]

**Pull request merged**